### PR TITLE
fix: wrap validator

### LINKED ISSUE #32: Allow user to define a config file without validation
**Is your feature request related to a problem? Please describe.**
If a user defines a config file for the session settings, the service will try to parse the config file. Since the service doesn't have direct access to VAST, we should allow a user to point to a session config file without validation.

**Describe the solution you'd like**
Add a wrap validator to allow the session settings to allow for just a single `user_settings_config_file` field.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by jtyoung84**
Closes #32 

- Adds wrap validator to allow SessionSettings to just point to a config file.

### CODE REVIEW

File: src/aind_data_transfer_models/core.py [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
 from enum import Enum
 from pathlib import PurePosixPath
-from typing import Any, ClassVar, List, Optional, Set, Union
+from typing import Any, ClassVar, List, Optional, Set, Union, get_args
 
 from aind_data_schema_models.data_name_patterns import build_data_name
 from aind_data_schema_models.modalities import Modality
 from aind_data_schema_models.platforms import Platform
+from aind_metadata_mapper.models import (
+    JobSettings as GatherMetadataJobSettings,
```
Comment (helen-m-lin) [inline src/aind_data_transfer_models/core.py:14]: nit/optional: there's 2 import blocks for  `from aind_metadata_mapper.models import (...`, they can be combined

File: src/aind_data_transfer_models/core.py [hunk 2]
```diff
@@ -276,40 +281,125 @@ def _parse_datetime(cls, datetime_val: Any) -> datetime:
...
+                    "user_settings_config_file": user_defined_session_settings[
+                        "job_settings"
+                    ]["user_settings_config_file"],
+                    "job_settings_name": user_defined_session_settings[
+                        "job_settings"
+                    ]["job_settings_name"],
+                }
+            )
+            validated_gather_configs.session_settings = session_settings
+            # user_defined_metadata_configs["session_settings"] = (
```
Comment (helen-m-lin) [inline src/aind_data_transfer_models/core.py]: nit: remove if not needed

File: src/aind_data_transfer_models/core.py [hunk 3]
```diff
@@ -276,40 +281,126 @@ def _parse_datetime(cls, datetime_val: Any) -> datetime:
...
+            ),
+            "procedures_settings": ProceduresSettings(
+                subject_id=validated_self.subject_id
+            ),
+            "raw_data_description_settings": RawDataDescriptionSettings(
+                name=validated_self.s3_prefix,
+                project_name=validated_self.project_name,
+                modality=([mod.modality for mod in validated_self.modalities]),
+            ),
+            # "metadata_dir": metadata_dir,
```
Comment (helen-m-lin) [inline src/aind_data_transfer_models/core.py]: nit: remove if unused

**Review by helen-m-lin — APPROVED** [review]
Added minor/optional suggestions. Everything looks good.

File: src/aind_data_transfer_models/core.py (hunk 1 above)
Comment (jtyoung84) [reply src/aind_data_transfer_models/core.py:14]: It looks like `isort` enforces this pattern

**Pull request merged**